GetAudio.cpp

```diff
@@ -1,6 +1,10 @@
 
 #include <thread>
 #include "GetAudio.h"
+#include <cassert>
+#include <iostream>
+#include <sstream>
+
 #define EXIT_ON_ERROR(hres)  \
               if (FAILED(hres)) { return false; }

```

Comment:
@Taha39  if u find time romove this macro as well and where it is used put the code there in with proper logging the message

Reply:
Corrected this.